themes/multi-step/composer.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "mautic/theme-multi-step",
+  "description": "Multi-step theme",
+  "type": "mautic-theme",
+  "keywords": ["mautic","theme"],
+  "minimum-stability": "dev",
+  "require": {
+    "mautic/core-lib": ">=6.0"
```

```suggestion
    "mautic/core-lib": ">=7.0"
```
If I remember correctly, the Twig function `getRoundedCorners` was introduced in Mautic 7, not Mautic 6.